src/yang/ietf-qos-action.yang

```diff
@@ -540,6 +543,24 @@ module ietf-qos-action {
       "Random Early Detect (RED) configuration parameters.";
   }
 
+  grouping codel-config-parameters {
+    leaf target {
+      type uint64;
+      default 5000;
+      description
+        "target time in microseconds spent by each packet in queue.";
+    }
+
+    leaf interval {
+      type uint64;
+      default 100000;
+      description
+        "The time in microsecond for minimum interval for congestion to persist before algorithm kicks in ";
```

Comment:
Same comment. Add a unit statement, and please wrap the description statement around column 70.